tests/integration/src/test/resources/org/mule/test/integration/exceptions/catch-exception-strategy-transaction-flow.xml

```diff
@@ -23,7 +23,7 @@
         <jms:inbound-endpoint queue="in1" connector-ref="activeMq">
             <jms:transaction action="ALWAYS_BEGIN"/>
         </jms:inbound-endpoint>
-        <test:component throwException="true"/>
```

Comment:
Is thrown using the callback since depending on the test case is has to fail or not. And also the TX is change at that point which is not possible if the exception is thrown using the attribute